analysis_driver/dataset.py

```diff
@@ -403,6 +433,31 @@ def number_of_samples(self):
                 raise AnalysisDriverError('Could not find number of quoted samples in LIMS for ' + self.name)
         return self._number_of_samples
 
+    @property
```

Comment:
ProjectDataset has to find its species a little differently from SampleDataset. These methods have been moved here from `pipelines.projects`